Tests/Tests/DatastoreCRUD.m

```diff
@@ -83,6 +83,41 @@ -(NSArray*)generateDocuments:(int)count
 
 #pragma mark - CREATE tests
 
+
+-(void) testDocumentWithInfinityValue
+{
+    NSError * error;
+    CDTMutableDocumentRevision * revision = [CDTMutableDocumentRevision revision];
+    revision.body = @{@"infinity":@(INFINITY)};
+    
+    CDTDocumentRevision * rev = [self.datastore createDocumentFromRevision:revision
+                                                                     error:&error];
+    
+    XCTAssertNil(rev,@"revision is not nil");
+    XCTAssertNotNil(error,@"Error should be set");
+    XCTAssertEqual(400,
```

Comment:
umm.. I see you made the error `kTDStatusBadJSON` above, but in my space it is:
```objc
    kTDStatusBadJSON = 493,
```
How does this test pass?